examples/debug_lines/main.rs

```diff
@@ -2,19 +2,16 @@
 
 extern crate amethyst;
 
-use amethyst::{
-    controls::{FlyControlBundle, FlyControlTag},
-    core::Time,
-    core::{
-        cgmath::{Deg, Point3, Vector3},
-        transform::{GlobalTransform, Transform, TransformBundle},
-    },
-    ecs::{Read, System, Write},
-    input::InputBundle,
-    prelude::*,
-    renderer::*,
-    utils::application_root_dir,
-};
+use amethyst::controls::{FlyControlBundle, FlyControlTag};
```

Comment:
Did rustfmt do this? It was literally changed a few days ago to the format you removed :stuck_out_tongue:

Reply:
Yep, it is rustmft `rustfmt 0.99.1-stable (da17b68 2018-08-04)`

Reply:
Well then. I guess @jojolepro was right after all. :sweat_smile:

Reply:
So, I can revert the particular changes.

Reply:
Why are they doing this?